lib/matplotlib/image.py

```diff
@@ -773,6 +778,23 @@ def set_interpolation(self, s):
         self._interpolation = s
         self.stale = True
 
+    def set_interpolation_stage(self, s):
+        """
+        Set whether interpolation happens after the image has been
+        transformed to RGBA.
```

Comment:
The 'stage' is more general than this implied, even if we only have two options
```suggestion
        Set when interpolation happens during the transform to RGBA.
```